### DISCUSSION TITLE
How can I enable transparency for the border_color attribute in the CTkButton class?

### DISCUSSION THREAD

**Discussion by elly-hacen** [TomSchimansky/CustomTkinter#2835] [Q&A]
`Image with corner radius:` 

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/55024568-29cc-46d9-b855-e023029f364f)

I'm using the `CTkButton` class from the `customtkinter` library, and I would like to make the button's border color transparent. However, when I set the `border_color` attribute to "transparent", I encounter a `ValueError` stating that transparency is not allowed for this attribute.

I would like to know if there is a way to enable transparency for the `border_color` attribute in the `CTkButton` class. If there is no built-in support for transparency, I would appreciate guidance on modifying the `CTkButton` class to allow transparency for the border color.

Any help or insights would be greatly appreciated. Thank you!

**Comment by ghost** [comment]
Next time, please ask questions in the Q&A section of the Discussions tab. If you find out this isn't possible through `customtkinter` and you want the feature (or you find a bug), then open an issue.

**Comment by ghost** [comment]
One work around would be to set the border color to the background color. This isn't ideal due to theme changes and light/dark mode.

Furthermore, I'd suggest changing the title to say "Add transparency for border color attribute" or something like that. The new title makes your issue a feature request/enhancement rather than a question.

**Comment by elly-hacen** [comment]
Apologies for any inconvenience caused. I am currently learning GitHub and custom tkinter, and my knowledge in these areas is limited at the moment. I will become proficient in these tools in the future. Thank you for your understanding and support.

**Comment by ghost** [comment]
No problem, in this case the issue is fine as you'd basically be asking for transparency to be added as a border color option. I'd suggest changing the title to make this more clear.

**Comment by Akascape** [comment]
@elly-hacen A new solution: https://github.com/TomSchimansky/CustomTkinter/discussions/2214
